index.html

```diff
@@ -1,160 +1,246 @@
+        <li>
+          <div class="lang_tech">
+            <h3 class="lang_title">Languages</h3>
+            <img class="lang_arrow" src="./image/Disabled.png" alt="" />
+          </div>
 
-        <ul class="about_group">
-            <li>
-                <div class="lang_tech">
-                    <h3 class="lang_title">Languages</h3>
-                    <img class="lang_arrow" src="./image/Disabled.png" alt="">
-                </div>
-                
-                <ul class="about_flex">
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/JS_icon.png" alt="">
-                        <h4 class="about_lang">JavaScript</h4>
-                    </li>
-    
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/html_icon.png" alt="">
-                        <h4 class="about_lang">HTML</h4>
-                    </li>
-    
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/css_icon.png" alt="">
-                        <h4 class="about_lang">CSS</h4>
-                    </li>
-    
-                    
-                </ul>
+          <ul class="about_flex">
+            <li class="about_skill">
+              <img class="about_img" src="./image/JS_icon.png" alt="" />
+              <h4 class="about_lang">JavaScript</h4>
             </li>
-            
 
-            <li class="lang_tech border">
-                <h3 class="lang_title">Frameworks</h3>
-                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            <li class="about_skill">
+              <img class="about_img" src="./image/html_icon.png" alt="" />
+              <h4 class="about_lang">HTML</h4>
             </li>
 
-            <li class="lang_tech border">
-                <h3 class="lang_title">Skills</h3>
-                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            <li class="about_skill">
+              <img class="about_img" src="./image/css_icon.png" alt="" />
+              <h4 class="about_lang">CSS</h4>
             </li>
+          </ul>
+        </li>
 
+        <li class="lang_tech border">
+          <h3 class="lang_title">Frameworks</h3>
+          <img class="lang_arrow" src="./image/narrow.png" alt="" />
+        </li>
 
-        </ul>
+        <li class="lang_tech border">
+          <h3 class="lang_title">Skills</h3>
+          <img class="lang_arrow" src="./image/narrow.png" alt="" />
+        </li>
+      </ul>
+    </section>
 
+    <section class="sectionForm">
+      <div class="form">
+          <h2 class="form-title">Contact me</h2>
+          <p class="form-parragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+        <form class="form-form" action="" method="POST">
+          <input class="formInput"
```

Comment:
- [ ] Kindly create a form using  [formspree service](https://formspree.io/html) and use the link generated in your form `action` so that your form can be submitted